src/student.rb

```diff
@@ -0,0 +1,14 @@
+require_relative './person'
+
+class Student < Person
+  attr_accessor :classroom
+
+  def initialize(age, classroom)
+    super age
+    @classroom = classroom
+  end
```

Comment:
- [ ] Kindly add name and parent_permission to initialization and pass them to the super method also, you can refer to this [lesson](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#inheritance.)
> It will be like this:
```ruby
  def initialize(age, classroom, name = "Unknown", parent_permission = true)  
    super(age, name, parent_permission)
    @classroom = classroom
  end
```